actioncable/README.md

```diff
@@ -526,6 +526,15 @@ cable.subscriptions.create 'AppearanceChannel',
     # normal channel code goes here...
 ```
 
+## Download and Installation
+
+The latest version of Action Cable can be installed with [RubyGems](#gem-usage),
```

Comment:
You said to rm this, no https://github.com/rails/rails/pull/25626#discussion_r69349788...?